base/doc/ltnews39.tex

```diff
@@ -187,6 +187,16 @@ \subsection{Warn if counter names are problematic}
 %
 \githubissue{823}
 
+\subsection{Optimise creation of simple document commands}
+
+Creating document commands using \cs{NewDocumentCommand}, etc., provides a very
+flexible way of grabbing arguments. When the document command only takes simple
+mandatory arguments, this has to-date added an overhead that could be avoided.
+We have now refined the internal code path such that \enquote{simple} document
+commands avoid any overhead at point-of-use, making the results as efficient as
+using \cs{newcommand} or low-level \TeX{} constructs.
```

Comment:
We could add something that explains that there is still a difference between \newcommand and \NewDocumentCommand as the former expands so corresponds to \NewExpandableDocumentCommand